src/ServerConfig.ts

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
+
+export class ServerConfig {
+    constructor(){
+    }
+
+    static loadConfigPath(){
+        return path.resolve(__dirname, '../server-config.json');
```

Comment:
I don't know its exact usage, but maybe it could be useful to load the configuration once and avoid reloading the file each time the method is called.

Reply:
this could be OK as long as this static method is only called once and its result stored somewhere (e.g. an object).

right now however, it seems like this method is not called at all in this repo